index.html

```diff
@@ -216,12 +252,24 @@ <h2>Computing resources etc.</h2>
                <input type="checkbox">
           </div>
           <div class="text">
-               Make sure you have a NERSC account and remember your password (<em>nope, a SSH key is not enough</em>) and have 2-step verification (MFA) set up!
-               <a href="https://confluence.slac.stanford.edu/x/mgRTD" title="link to Confluence">Need to apply for one?</a>
-               <a href="https://nim.nersc.gov/forgotpass.php">Forget your password?</a>
-               Questions? Ask them in the
-               <a href="https://lsstc.slack.com/messages/C2U2K05JR/" title="link to LSSTC Slack">#desc-nersc</a> Slack
-               channel. Also, make sure you are in the "lsst" group. If not, contact Richard Dubois or Debbie Bard to request to be added to the "lsst" group.
+               Make sure you have a NERSC account and remember your password
+               (<em>nope, a SSH key is not enough</em>) and have 2-step verification (MFA) set up!
+               <ul>
+                    <li>
+                    <a href="https://confluence.slac.stanford.edu/x/mgRTD"
+                    title="link to Confluence">Need to apply for one?</a>
+                    </li>
+                    <li>
+                    <a href="https://nim.nersc.gov/forgotpass.php">Forgot your password?</a>
+                    </li>
+                    <li>
+                    Questions? Ask them in the
+                    <a href="https://lsstc.slack.com/messages/C2U2K05JR/"
+                    title="link to LSSTC Slack">#desc-nersc</a> Slack channel.
+                    </li>
+               </ul>
+               Also, make sure you are in the "lsst" group. If not, contact
+               Richard Dubois or Debbie Bard to request to be added to the "lsst" group.
```

Comment:
I suggest indicating how to contact Richard or Debbie (e.g., suggest using slack and give their slack handles).

Reply:
Also I just checked in with Richard because confluence said something slightly different.  He told me that both Debbie Bard and Heather Kelly are co-PIs on the allocation, so actually all 3 of them can be listed as contacts here.